### DISCUSSION TITLE
SAML for delegated authentication

### DISCUSSION THREAD

**Discussion by vermorel** [plausible/analytics#328] [Anything Else]
For a corporate use, delegated authentication is really a must-have. You don't want people having to manage yet another password. In particular, as most companies either operate with Office 365 or Google Apps (or both), it would be really nice to support SAML for delegated authentication.

Disclaimer: I operate a B2B SaaS business, and the vast majority of our clients already have a IT system in place that support SAML.

**Comment by TechupBusiness** [comment]
I would also need this feature... SAML 2.0

**Comment by wphillips** [comment]
yes please SAML/OKTA support.  Will be hard to get security to agree to this solution without it.
